tbx/project_styleguide/templates/patterns/pages/team/team_listing.html

```diff
@@ -2,12 +2,13 @@
 {% load wagtailcore_tags wagtailimages_tags static %}
 
 {% block content %}
-    {% include "patterns/molecules/title-filters/title-filters.html" with title=page.strapline tags=tags item=page %}
+
+    {% include "patterns/molecules/title-filters/title-filters.html" with tags=tags item=page %}
+
+    {{ page.strapline }}
```

Comment:
**Comment:** This renders the `span` tag in the page strapline as text, it might need making safe if it is this similar to production data.

Reply:
Good  spot, thanks, will update

Reply:
Updated in 0436e30

Reply:
We no longer need to render spans inside the strapline - this was for the old site where span tags were displayed in a different colour. Not sure if that makes a difference to the changes you've made @engineervix

Reply:
Removed in 205022a